src/main/java/com/powsybl/openloadflow/network/impl/LfDanglingLineBranch.java

```diff
@@ -7,10 +7,7 @@
  */
 package com.powsybl.openloadflow.network.impl;
 
-import com.powsybl.iidm.network.DanglingLine;
-import com.powsybl.iidm.network.LimitType;
-import com.powsybl.iidm.network.LoadingLimits;
-import com.powsybl.iidm.network.TwoSides;
+import com.powsybl.iidm.network.*;
```

Comment:
Did you remove the explicit imports on purpose ?
Or is it in anitelliJ missconfiguration ? In this case this post may help https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Reply:
Closing since this has been fixed.